src/main/java/io/appium/java_client/service/local/AppiumDriverLocalService.java

```diff
@@ -219,4 +233,132 @@ public void addOutPutStreams(List<OutputStream> outputStreams) {
     public boolean clearOutPutStreams() {
         return stream.clear();
     }
+
+    /**
+     * Enables server output data logging through
+     * <a href="http://slf4j.org">SLF4J</a> loggers. This allow server output
+     * data to be configured with your preferred logging frameworks (e.g.
+     * java.util.logging, logback, log4j).
+     * 
+     * <p>NOTE1: You might want to call method {@link #clearOutPutStreams()} before
+     * calling this method.<br>
+     * NOTE2: it is required that {@code --log-timestamp} server flag is
+     * {@code false}.
+     * 
+     * <p>By default log messages are:
+     * <ul>
+     * <li>logged at {@code INFO} level, unless log message is pre-fixed by
+     * {@code [debug]} then logged at {@code DEBUG} level.</li>
+     * <li>logged by a <a href="http://slf4j.org">SLF4J</a> logger instance with
+     * a name corresponding to the appium sub module as prefixed in log message
+     * (logger name is transformed to lower case, no spaces and prefixed with
+     * "appium.service.").</li>
+     * </ul>
+     * Example log-message: "[ADB] Cannot read version codes of " is logged by
+     * logger: {@code appium.service.adb} at level {@code INFO}.
+     * <br>
+     * Example log-message: "[debug] [XCUITest] Xcode version set to 'x.y.z' "
+     * is logged by logger {@code appium.service.xcuitest} at level
+     * {@code DEBUG}.
+     * <br>
+     * 
+     * @see #addSlf4jLogMessageConsumer(BiConsumer)
+     */
+    public void enableDefaultSlf4jLoggingOfOutputData() {
+        addSlf4jLogMessageConsumer((logMessage, ctx) -> {
+            if (ctx.getLevel().equals(DEBUG)) {
```

Comment:
this is anyway not very accurate, since this `[debug]` prefix is also not always added to debug logs

Reply:
@mykola-mokhnach So what do you sugests instead? Should all be logged at ÌNFO` level?
As it is now log messages with `[debug]` prefix is logged at `DEBUG` level. That is better than nothing if you ask me.